packages/design-system/src/components/OcTile/OcTile.vue

```diff
@@ -93,8 +95,8 @@ export default defineComponent({
 
 <style lang="scss" scoped>
 .oc-tile-card {
-  box-shadow: none !important;
   background-color: var(--oc-color-background-highlight) !important;
+  box-shadow: 3px 0 10px rgb(0 0 0 / 15%);
```

Comment:
Please keep the box-shadow as `none` like before, but add `outline: 1px solid var(--oc-color-border);` instead. Discussed it with @tbsbdr that's how we want to start. Already had too much harsh feedback with shadows in other places of oC Web. 😅